apps/admin-web/src/index.tsx

```diff
@@ -1,7 +1,50 @@
-import * as React from "react";
-import * as ReactDOM from "react-dom";
+import React from "react";
+import ReactDOM from "react-dom";
+import { setGlobal } from "reactn";
+
+import { ApolloProvider } from "@apollo/react-hooks";
+import { InMemoryCache } from "apollo-cache-inmemory";
+import ApolloClient from "apollo-client";
+import { setContext } from "apollo-link-context";
+import { createHttpLink } from "apollo-link-http";
+
 import { App } from "./App";
 import registerServiceWorker from "./registerServiceWorker";
 
-ReactDOM.render(<App />, document.getElementById("root") as HTMLElement);
+import { config } from "./config";
+
+setGlobal({
+  events: [],
+  eventFormVisible: false,
+  activeEvent: undefined
+});
+
+const httpLink: any = createHttpLink({
+  uri: config.GRAPHQL_URL
+});
+
+const authLink: any = setContext((_: any, { headers }: any) => {
+  // get the authentication token from local storage if it exists
+  const token: string | null = localStorage.getItem("token");
```

Comment:
Can you store this name in an config variable just in case other things need to grab it.